PatchesAndAreasApi.Tests/V1/Gateways/DynamoDbGatewayTests.cs

```diff
@@ -67,6 +67,18 @@ private UpdatePatchesResponsibilitiesRequestObject ConstructUpdateRequest(Guid r
             return request;
         }
 
+        private (PatchEntity, List<ResponsibleEntities>) CreateEntityWithResponsibleEntities()
```

Comment:
Small tip, you can use params instead of a list. That means you can either pass one, or a list.